### PR TITLE
fix: small additions/changes to the chart release flow CY-5281

### PR DESCRIPTION
**Pull request by pedrobpereira**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -86,7 +86,9 @@ The Release Manager must create a release candidate branch:
 
     -   Create a `.version` file based on the most recent tag that you created on the previous step.
 
-    -   Update the `requirements.lock` with the latest versions and freeze the `global.workers.config.imageVersion` version on `./codacy/values.yaml`.
+    -   Update the `requirements.lock` with the latest versions, manually copy the versions to the `requirements.yaml` file, this ensures a lock on versions and facilitates the creation of release candidates.
```
Comment (prcr) [inline RELEASE.md:89]: Suggestion to split sentence to improve readability:

```suggestion
    -   Update the `requirements.lock` with the latest versions and manually copy the versions to the `requirements.yaml` file. This ensures a lock on versions and facilitates the creation of release candidates.
```

File: RELEASE.md [hunk 2]
```diff
@@ -86,7 +86,9 @@ The Release Manager must create a release candidate branch:
...
+    
+        Freeze the `global.workers.config.imageVersion` version on `./codacy/values.yaml`.
```
Comment (prcr) [inline RELEASE.md:91]: Consider including this instruction on a separate step:

```suggestion
    -   Freeze the `global.workers.config.imageVersion` version on `./codacy/values.yaml`.
```

File: RELEASE.md [hunk 3]
```diff
@@ -250,11 +252,9 @@ Then, the Release Manager releases and announces the new version:
          git push --tag origin self-hosted-x.x.x
          ```
 
--   [ ] 3.  Inform all stakeholders that the release is finished
-
     The final version will be `x.x.x`.
 
--   [ ] 4. Finally, the Technical Writer must [release a version of the documentation for the new Codacy Self-hosted version](https://github.com/codacy/docs/blob/master/CONTRIBUTING.md#releasing-a-new-version-of-the-documentation).
+-   [ ] 3. Finally, the Technical Writer must [release a version of the documentation for the new Codacy Self-hosted version](https://github.com/codacy/docs/blob/master/CONTRIBUTING.md#releasing-a-new-version-of-the-documentation).
```
Comment (prcr) [inline RELEASE.md:257]: Extra indentation space just to keep the formatting consistent. :nerd_face: 

```suggestion
-   [ ] 3.  Finally, the Technical Writer must [release a version of the documentation for the new Codacy Self-hosted version](https://github.com/codacy/docs/blob/master/CONTRIBUTING.md#releasing-a-new-version-of-the-documentation).
```

**Review by prcr — COMMENTED** [review]
I left a few suggestions. Thanks for helping improve the process!

File: RELEASE.md (hunk 3 above)
Comment (pedrocodacy) [reply RELEASE.md:265]: @prcr we had to close this PR and open a new one (we did a mistake on branch name and were triggering the wrong CI build). This is open in #707 (we will apply your comments anyways)

Comment (prcr) [reply RELEASE.md:265]: Yes, I also have to be careful not to open branches starting with `release` on the docs repository! :sweat_smile:

**Pull request closed without merging**